### PR TITLE
docs: pin a specific version of the Twilio CLI

### PR DESCRIPTION
**Pull request by thinkingserious**
A short description of what this PR does.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: docs/general_usage.md [hunk 1]
```diff
@@ -115,6 +115,22 @@ ps -aux | twilio email:send \
   --text="See attachment"
 ```
 
+## Troubleshooting
+
+### Installing a specific version of the Twilio CLI
```
Comment (childish-sambino) [inline docs/general_usage.md]: `s/the Twilio CLI/twilio-cli/`

File: docs/general_usage.md [hunk 2]
```diff
@@ -115,6 +115,22 @@ ps -aux | twilio email:send \
...
+
+General installation instructions are located [here](https://www.twilio.com/docs/twilio-cli/quickstart#install-twilio-cli).
```
Comment (childish-sambino) [inline docs/general_usage.md]: Prefer short URL: `https://twil.io/cli#install-twilio-cli`

File: docs/general_usage.md [hunk 3]
```diff
@@ -115,6 +115,22 @@ ps -aux | twilio email:send \
...
+
+A list of tagged releases can be found with `git ls-remote --tags https://github.com/twilio/twilio-cli.git`.
+
+#### Mac OSX
```
Comment (childish-sambino) [inline docs/general_usage.md]: `s/OSX/OS X/`

File: docs/general_usage.md [hunk 4]
```diff
@@ -115,6 +115,22 @@ ps -aux | twilio email:send \
...
+
+`brew install https://raw.githubusercontent.com/twilio/homebrew-brew/X.X.X/Formula/twilio.rb`
+
+#### Windows / Linux
+
+`npm install twilio-cli@X.X.X -g`
+
```
Comment (childish-sambino) [inline docs/general_usage.md]: Need instructions on how to uninstall the existing installation.

File: docs/general_usage.md [hunk 5]
```diff
@@ -115,6 +115,22 @@ ps -aux | twilio email:send \
...
+### Installing a specific version of the Twilio CLI
+
+General installation instructions are located [here](https://www.twilio.com/docs/twilio-cli/quickstart#install-twilio-cli).
+
+A list of tagged releases can be found with `git ls-remote --tags https://github.com/twilio/twilio-cli.git`.
+
+#### Mac OSX
+
+`brew install https://raw.githubusercontent.com/twilio/homebrew-brew/X.X.X/Formula/twilio.rb`
```
Comment (childish-sambino) [inline docs/general_usage.md]: Need verbiage that indicates the `X.X.X` needs to be replaced.

**Comment by childish-sambino** [comment]
Need to merge in master.

**Comment by childish-sambino** [comment]
Need to rebase.

File: docs/general_usage.md [hunk 6]
```diff
@@ -115,6 +115,26 @@ ps -aux | twilio email:send \
...
+
+General installation instructions are located [here](https://twil.io/cli#install-twilio-cli).
+
+A list of tagged releases can be found with `git ls-remote --tags https://github.com/twilio/twilio-cli.git`.
+
+Substitute X.X.X below with the desired version.
+
+#### Mac OS X
+
+`brew install https://raw.githubusercontent.com/twilio/homebrew-brew/X.X.X/Formula/twilio.rb`
```
Comment (childish-sambino) [inline docs/general_usage.md]: Need uninstall steps for brew.

Comment (thinkingserious) [reply docs/general_usage.md]: Those steps are given when you run that command by homebrew itself, if it's necessary.

Comment (childish-sambino) [reply docs/general_usage.md]: Ack.

**Pull request merged**